templates/argoCD-diff-pr-comment.gotmpl

```diff
@@ -1,4 +1,7 @@
 {{define "argoCdDiff"}}
+{{ if .Header }}
+{{ .Header }}
```

Comment:
I think it would force  use to add more whitespace control characters to avoid actually adding the extra spaces.
And those are hard to understand by themselves